app/src/main/java/ar/com/wolox/android/example/ui/newdetail/NewDetailPresenter.kt

```diff
@@ -0,0 +1,79 @@
+package ar.com.wolox.android.example.ui.newdetail
+
+import android.app.Application
+import ar.com.wolox.android.example.model.New
+import ar.com.wolox.android.example.network.NewsService
+import ar.com.wolox.android.example.utils.NetworkUtils
+import ar.com.wolox.android.example.utils.UserSession
+import ar.com.wolox.android.example.utils.networkCallback
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import ar.com.wolox.wolmo.networking.retrofit.RetrofitServices
+import javax.inject.Inject
+
+class NewDetailPresenter @Inject constructor(private val mRetrofitServices: RetrofitServices, private val mUserSession: UserSession, private val mApplication: Application) : BasePresenter<INewDetailView>() {
```

Comment:
Move parameters (retrofit and userSession) to new lines. Avoid too long lines. Moreover, remember to remove m as prefix.